packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -224,6 +224,11 @@ def get_configuration_parameters_with_type_and_default_values(
             else:
                 default_value = f"Path({default_value})"
 
+        if metadata.message_type and metadata.message_type == "ni.protobuf.types.DoubleXYData":
```

Comment:
What message types are supported? Should this error for any "truthy" message_type?

Reply:
No message types are supported, as only `DoubleXYData` has the type `type_pb2.Field.TYPE_MESSAGE`. To be more specific for `DoubleXYData` type, I added the second check. I will remove that.

Reply:
I have updated the if condition to check "truthy" message_type.